app/Widgets/Literacy/Migrations/2021_01_15_102625_create_literacy_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateLiteracyTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('literacy_rates', function (Blueprint $table) {
+            $table->id();
+            $table->bigInteger('total');
```

Comment:
Since your data uses decimals, I think it would be better to show some of the decimal points.  Can you change the total field to:

```
$table->decimal('total', 8, 3);
```

If you want to test it locally, you will need to do the following:

1. Roll back the database: `php artisan migrate:rollback --step=1`
2. Make your change.
3. Migrate the database: `php artisan migrate`
4. Import your data.

Now all the fields should be up to 3 decimal points.